tests/git-notes.test.ts

```diff
@@ -53,15 +53,20 @@ test("set/get notes", async () => {
     expect(await notes.getLastCommitNote(commit as string)).toEqual("2");
 
     // error tests for update
-    await expect(notes.update(gitURL)).resolves.toBeUndefined();
+    for (const note of ["commit-to-mail", "mail-to-commit"]) {
```

Comment:
Wasn't the network already being avoided by using `fakeRemote` instead of `gitURL` in PR #1786?

Reply:
You're correct! I guess I forgot about that when I rebased the branch. Will drop https://github.com/gitgitgadget/gitgitgadget/pull/1473/commits/615307bc55eab859e832aca55ec257fd7aebdd86 from this here PR.